app/vmui/packages/vmui/src/components/Chart/GraphTips/contants/tips.tsx

```diff
@@ -51,15 +51,16 @@ const legendTips = [
     </>
   },
   {
-    title: "Copy key-value pairs",
+    title: "Copy label key-value pairs",
```

Comment:
nit: label it is a key label pair

Reply:
I think it is OK to say `label key-value pair`